managed/resource_cluster_db_audit_export_config.go

```diff
@@ -0,0 +1,394 @@
+package managed
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	retry "github.com/sethvargo/go-retry"
+	openapiclient "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+)
+
+type resourceClusterDbAuditExportConfigType struct{}
+
+// DefaultTrueModifier sets the default value of a boolean attribute to true if not set.
+type DefaultTrueModifier struct{}
+
+func (m DefaultTrueModifier) Description(ctx context.Context) string {
+	return "If not configured, defaults to true."
+}
+
+func (m DefaultTrueModifier) MarkdownDescription(ctx context.Context) string {
+	return "If not configured, defaults to `true`."
+}
+
+func (m DefaultTrueModifier) Modify(ctx context.Context, req tfsdk.ModifyAttributePlanRequest, resp *tfsdk.ModifyAttributePlanResponse) {
```

Comment:
Why do you need that ?

Reply:
This method is mandatory part of plan modifier interface

Reply:
Removed